distribution/src/main/assembly/assembly-openshift-drools-hacep.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<assembly xmlns="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:schemaLocation="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.2 http://maven.apache.org/xsd/assembly-1.1.2.xsd">
+
+  <id>assembly-openshift-drools-hacep</id>
+  <formats>
+    <format>dir</format>
+    <format>zip</format>
+  </formats>
+
+  <includeBaseDirectory>true</includeBaseDirectory>
+
+  <fileSets>
+    <fileSet><!-- Note: going outside the module dir is bad, but it is not fetching generated files -->
+      <includes>
+        <include>../LICENSE-Apache-2.0.txt</include>
+      </includes>
+      <outputDirectory/>
+    </fileSet>
+    <!-- Examples -->
+    <fileSet><!-- Note: going outside the module dir is bad, but it is not fetching generated files -->
+      <directory>..</directory>
+      <outputDirectory>sources</outputDirectory>
+      <excludes>
+        <exclude>**/target/**</exclude>
+        <exclude>**/local/**</exclude>
+        <exclude>.*/**</exclude>
+        <exclude>nbproject/**</exclude>
+        <exclude>*.ipr</exclude>
+        <exclude>*.iws</exclude>
+        <exclude>**/*.iml</exclude>
+        <exclude>.git/**</exclude>
+        <exclude>/distribution/**</exclude>
```

Comment:
You need to update root `pom.xml` file too to properly remove a module, it is not enough to exclude it here. Otherwise the target source will not compile.
I think the simplest way is to add a profile to root `pom.xml` with the reduced list of modules and document that dist sources can be built only with that specific profile

Reply:
@danielezonca why not to leave the distribution module in the zip? (I mean, not exclude it)

Reply:
@Ginxo the comment is not related to that module in particular but more in general: as soon as we exclude even one module, we need to update root pom.xml too otherwise it will not build (from dist/source folder).
Btw I think it has not real sense to have distribution module in the distribution itself :)

Reply:
@mareknovotny @danielezonca this is happening in every distribution module we have, so I would create a reengineering task just for excluding the distribution module from every distribution file